webhooks/2025-04-15.yml

```diff
@@ -2536,9 +2541,16 @@ components:
         payment_method:
           type: string
           description: Type of payment method (e.g., "card")
+          enum:
```

Comment:
To confirm, these are the only two `payment_method` types we allow to be used with our C1's Stripe Connected accounts?

Reply:
That is my understanding; we may expand this list in the future, but at the moment those are the only two values that exist.